ui/src/dashboards/components/AxesOptions.js

```diff
@@ -46,49 +49,75 @@ const AxesOptions = ({
             type="number"
           />
         </div>
-        {/* <div className="form-group col-sm-6">
-          <label htmlFor="prefix">Labels Prefix</label>
-          <input
-            className="form-control input-sm"
-            type="text"
-            name="prefix"
-            id="prefix"
+        <Input
+          name="prefix"
+          id="prefix"
+          value={prefix}
+          labelText="Y-Value's Prefix"
+          onChange={onSetPrefixSuffix}
+        />
+        <Input
+          name="suffix"
+          id="suffix"
+          value={suffix}
+          labelText="Y-Value's Suffix"
+          onChange={onSetPrefixSuffix}
+        />
+        <Tabber
+          labelText="Y-Value's Format"
+          tipID="Y-Values's Format"
+          tipContent={TOOLTIP_CONTENT.FORMAT}
+        >
+          <Tab
+            text="K/M/B"
+            isActive={base === BASE_10}
+            onClickTab={onSetBase(BASE_10)}
           />
-        </div>
-        <div className="form-group col-sm-6">
-          <label htmlFor="prefix">Labels Suffix</label>
-          <input
-            className="form-control input-sm"
-            type="text"
-            name="suffix"
-            id="suffix"
+          <Tab
+            text="K/M/G"
+            isActive={base === BASE_2}
+            onClickTab={onSetBase(BASE_2)}
           />
-        </div>
-        <div className="form-group col-sm-6">
-          <label>Labels Format</label>
-          <ul className="nav nav-tablist nav-tablist-sm">
-            <li className="active">K/M/B</li>
-            <li>K/M/G</li>
-          </ul>
-        </div>
-        <div className="form-group col-sm-6">
-          <label>Scale</label>
-          <ul className="nav nav-tablist nav-tablist-sm">
-            <li className="active">Linear</li>
-            <li>Logarithmic</li>
-          </ul>
-        </div> */}
+        </Tabber>
+        <Tabber labelText="Scale">
+          <Tab
+            text="Linear"
+            isActive={scale === LINEAR}
+            onClickTab={onSetScale(LINEAR)}
+          />
+          <Tab
+            text="Logaritmic"
+            isActive={scale === LOG}
+            onClickTab={onSetScale(LOG)}
+          />
+        </Tabber>
       </form>
     </div>
   )
 }
 
 const {arrayOf, func, shape, string} = PropTypes
 
+AxesOptions.defaultProps = {
+  axes: {
+    y: {
+      bounds: ['', ''],
+      prefix: '',
+      suffix: '',
+      base: BASE_10,
+      scale: LINEAR,
+      defaultYLabel: '',
```

Comment:
Is this being used? Because I don't see it being used, and further, the label is not being set to a default, which causes a `label` error in the Dygraph component.